man/runc-restore.8.md

```diff
@@ -75,6 +75,14 @@ daemon. See [criu --lazy-pages option](https://criu.org/CLI/opt/--lazy-pages).
 **apparamor** or **selinux**, and _label_ is a valid LSM label. For example,
 **--lsm-profile "selinux:system_u:system_r:container_t:s0:c82,c137"**.
```

Comment:
Not really related, but should we amend the doc to say something like "By default, the checkpointed LSM profile is used upon restore"?